internal/context.go

```diff
@@ -106,6 +114,36 @@ func ClearContext() string {
 	return previous
 }
 
+// LoginContext login to a Qlik Sense Enterprise and sets the X-Qlik-Session as a cookie
+func LoginContext(tlsClientConfig *tls.Config, contextName string, userName string, password string) {
+	handler := NewContextHandler()
+	context := handler.Get(contextName)
+
+	if context == nil {
+		log.Fatalf("context '%s' wasn't found.\n", contextName)
+	}
+
+	qlikSession := getSessionCookie(tlsClientConfig, context.Engine, userName, password)
+
+	if _, ok := context.Headers["cookie"]; ok {
+		// Cookie header present
+		re := regexp.MustCompile(`X-Qlik-Session=[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
```

Comment:
Isn't it enough with checking if a cookie `X-Qlik-Session` is set? Think the regex adds complexity.

Reply:
Only a novice would complain 🤔🧙‍♂️

Reply:
We are all novices when it comes to regex